doc/main/examples_testing/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2025 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.11)
+project(doc_examples_testing LANGUAGES CXX)
+
+set(_reference_examples_path "${CMAKE_CURRENT_SOURCE_DIR}/../reference/examples")
+set(_userguide_examples_path "${CMAKE_CURRENT_SOURCE_DIR}/../tbb_userguide/examples")
+
+add_custom_target(build-doc-examples
+    COMMENT "Build oneTBB documentation samples")
+set(doc_examples_test_label "doc-examples")
+
+macro(add_doc_example _doc_example_path)
+    message(STATUS "Adding ${_doc_example_path}")
```

Comment:
I would prefer not to output each added test during configuration. Maybe we should just output once that "Enabled testing of samples from documentation"?

Reply:
Done